### DISCUSSION TITLE
How to use halide to realize Linear Convolution operation

### DISCUSSION THREAD

**Discussion by wuhongsheng** [halide/Halide#7327] [Q&A]
How to use halide to realize Linear Convolution operation

**Comment by dkurt** [comment]
There are examples for 2D convolutions in [Halide](https://github.com/halide/Halide/blob/b1ca3345b81b3a5ebee16278d1a603b4fe7eae31/apps/conv_layer/conv_layer_generator.cpp#L25) and  [OpenCV](https://github.com/opencv/opencv/blob/fc27a343e986bfc7ecd5494940ee27bb73e155cd/modules/dnn/src/layers/convolution_layer.cpp#L770-L771). So you can adopt for 1D by simply merging spatial dimensions `x` and `y`.
